lib/rubocop/cop/rails/add_column_comment.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for migrations using `add_column` without passing a
+      # comment option.
+      #
+      # @example
+      #   # bad (no comment for a new column)
+      #   add_column :table, :column, :integer
+      #
+      #   create_table :table do |t|
+      #     t.type :column
+      #   end
+      #
+      #   # good
+      #   add_column :table, :column, :integer, comment: 'Number of offenses'
+      #
+      #   create_table :table do |t|
+      #     t.type :column, comment: 'Number of offenses'
+      #   end
+      #
+      class AddColumnComment < Base
+        include ActiveRecordMigrationsHelper
+
+        MSG = '`add_column` without the `comment` option.'
```

Comment:
```suggestion
        MSG = 'New column added without the `comment` option.'
```
(so as not be overly specific about the syntax)